static/demo.html

```diff
@@ -23,6 +23,12 @@
 
         <script src="demo/credentials.js"></script>
         <script src="demo/app.js"></script>
+
+        <style>
+            .ffwdme-debug-wrapper>button {
```

Comment:
Please move this to https://github.com/ffwdme/ffwdme.js/blob/master/src/debug/debug.scss

Reply:
Done